src/responder/nss/nsssrv.c

```diff
@@ -57,22 +57,24 @@ nss_clear_memcache(TALLOC_CTX *mem_ctx,
 {
     int memcache_timeout;
     errno_t ret;
+    int flag_fd;
 
-    ret = unlink(SSS_NSS_MCACHE_DIR"/"CLEAR_MC_FLAG);
-    if (ret != 0) {
+    flag_fd = open(SSS_NSS_MCACHE_DIR"/"CLEAR_MC_FLAG, O_RDONLY);
```

Comment:
Perhaps `stat()` (instead of `open()`) would be a little bit more convenient here.

Reply:
I've updated to use `access(.., F_OK)` since that does exactly what we need.